securedrop/journalist_app/utils.py

```diff
@@ -525,8 +525,13 @@ def col_download_all(cols_selected: List[str]) -> werkzeug.Response:
 
 def serve_file_with_etag(db_obj: Union[Reply, Submission]) -> flask.Response:
     file_path = Storage.get_default().path(db_obj.source.filesystem_id, db_obj.filename)
+    add_range_headers = not current_app.config["USE_X_SENDFILE"]
```

Comment:
A comment explaining why we're doing this might be nice so we don't forget.

Reply:
To avoid pushing further code changes I just beefed up the commit message. Please restamp if all looks well.